### PR TITLE
Add Hong Kong Principal Officials

### PR DESCRIPTION
**Pull request by atehe**
closes [Hong Kong: Principal Officials](https://github.com/orgs/opensanctions/projects/2/views/1?pane=issue&itemId=49554653)

### CODE REVIEW

File: datasets/hk/po/crawler.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    doc = context.fetch_html(url, cache_days=1)
+
+    member_header = get_element_text(doc, '//h1[contains(@class,"title")]')
+    member_desc = get_element_text(doc, '//section[@class="blockItem"]')
+
+    person_name = member_header.split(",")[0]
+    person_title = person_name.split()[0]
+    person_name = " ".join(person_name.split()[1:])
+
+    position_name = member_header.split(",")[-1]
```
Comment (jbothma) [inline datasets/hk/po/crawler.py:33]: With this, does `Secretary for Culture, Sports and Tourism` become `Sports and Tourism`?

I wonder if a better way might be 

1. take member_header[1:] (perhaps tell split to only split once)
2. delete all acronymns, e.g. `re.sub("([A-Z], )*[A-Z]", "", str)` (there are bugs in that, just the idea)
3. delete all acronymns

File: datasets/hk/po/crawler.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    position_name = f"{section} - {position_name.strip()}"
```
Comment (jbothma) [inline datasets/hk/po/crawler.py:34]: mmm I think `Secretary for Culture, Sports and Tourism` is a perfect position name.
I think `Directors of Bureaux - Secretary for Culture, Sports and Tourism` is more than we need. So let's rather not prefix the position name with it.

You could stick `Directors of Bureaux` in to the position description while you have it at hand.

**Review by jbothma — CHANGES_REQUESTED** [review]
Thank you! Just some small comments

File: datasets/hk/po/crawler.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    for string in to_remove:
+        element_text = element_text.replace(string, "")
+
+    return collapse_spaces(element_text.strip())
+
+
+def crawl_members(context: Context, section: str, elem: ElementTree):
+
+    url = elem.get("href")
+    url = context.data_url.replace("index.htm", url)
```
[the remark this answers is not in the document]
Comment (atehe) [reply datasets/hk/po/crawler.py:22]: thanks

File: datasets/hk/principal_officials/hk_principal_officials.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name: hk_principal_officials
+title: Principal Officals of Hong Kong
```
Comment (jbothma) [inline datasets/hk/principal_officials/hk_principal_officials.yml:2]: Hong Kong Principal Officials
```suggestion
title: Hong Kong Principal Officials
```

**Pull request merged**